docs/TileLink-Diplomacy-Reference/index.rst

```diff
@@ -0,0 +1,29 @@
+TileLink and Diplomacy Reference
+================================
+
+TileLink is the cache coherence and memory protocol used by RocketChip and
+other Chipyard generators. It is how different modules like caches, memories,
+peripherals, and DMA devices communicate with each other.
+
+RocketChip's TileLink implementation is built on top of Diplomacy, a framework
+for exchanging configuration information among Chisel generators in a two-phase
+elaboration scheme.
+
+A brief overview of how to connect simple TileLink widgets can be found
+in the :ref:`Adding-an-Accelerator` section. This section will provide a
+detailed reference for the TileLink and Diplomacy functionality provided by
```

Comment:
Is this a good place to talk about the GraphML file that helps visualize the diplomacy graph? @a0u